### PR TITLE
Issue #DP-1163 fix: There is lag on displaying the entered text in the search bar while fetching user

### PR DESCRIPTION
**Pull request by santoshilimi**

### CODE REVIEW

**Review by twsvaishali — APPROVED** [review]

File: src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.ts [hunk 1]
```diff
@@ -146,7 +146,17 @@ export class OrgUserListComponent implements OnInit, AfterViewInit {
     this.sortColumn = column;
   }
 
+  setTelemetryForonRoleChange(user) {
```
Comment (venkanagouda-k) [inline src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.ts:149]: Better to use telemetry directives.
These directives acts common channel to all interact events

File: src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.html [hunk 2]
```diff
@@ -68,18 +68,15 @@ <h4 class="sb-pageSection-title">{{ resourceService.frmelmnts.lbl.allUsers }}</h
...
-                        [telemetryInteractEdata]="getTelemetryInteractEdata('assign-role-by-contributingOrg', 'click', 'org-user-list', {values: [orgUser.identifier, option.name]})"
-                        [telemetryInteractObject]="telemetryInteractObject" [telemetryInteractCdata]="telemetryInteractCdata"
-                        [telemetryInteractPdata]="telemetryInteractPdata">
-                  </sui-select-option>
-                </sui-select>
-              </div>
+              <div class="custom-selectbx">
+                <label>
+                  <select class="custom-selectbx" [(ngModel)]="orgUser.selectedRole" labelField="name"  valueField="name" (change)="onRoleChange(orgUser)" >
+                    <option *ngFor="let option of roles;" >
```
Comment (venkanagouda-k) [inline src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.html:74]: *ngFor="let option of orgRole.filteredOptions"
Please check again, there is change in variable

Comment (twsvaishali) [reply src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.html:74]: @venkanagouda-k sui-select behaves in that way. as select element is tagged as #orgRole, the options were available in orgRole.filteredOptions

File: src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.ts (hunk 1 above)
Comment (santoshilimi) [reply src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.ts:149]: @venkanagouda-k as we are using plane html code here so i have implemented in ts file

File: src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.html (hunk 2 above)
Comment (santoshilimi) [reply src/app/client/src/app/modules/contribute/components/org-user-list/org-user-list.component.html:74]: yes

**Pull request merged**